classes/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(age, specialization)
+    super(age, name, parent_permission: true)
+    @specialization = specialization
```

Comment:
- [ ] You should add a `name` as a parameter here, since if you want to give a teacher a name other than the default, you need to be able to provide it in the initializer and pass it down to the super.